### PR TITLE
Local storage branch

### PR DESCRIPTION
**Pull request by toddtrivisonno**

### CODE REVIEW

File: src/Components/Card.js [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
+import './Card.css';
+
 
 class Card extends React.Component {
 
    constructor(props) {
       super(props);
       this.cardStyle = {
-         fontFamily: "cursive",
+         // fontFamily: "cursive",
```
Comment (JTHall94) [inline src/Components/Card.js:11]: Is this being kept for a future usage? If not, you may want to remove it. Seems like you're handling the font-styles efficiently in the css files.

File: src/Components/NavigationCard.js [hunk 2]
```diff
@@ -8,31 +8,36 @@ class NavigationCard extends React.Component {
...
       super(props);
-      // this.coursesArray = ['Antipasti', 'Primi', 'Secondi', 'Formaggi', 'Dolce'];
       this.cardStyle = {
-         fontFamily: "cursive",
+         // fontFamily: "cursive",
       }
+
    }
+   
+   createTab(label) {
```
Comment (JTHall94) [inline src/Components/NavigationCard.js:17]: This is a really cool way to handle generating the tabs. Nicely done!

File: src/Components/NavigationCard.css [hunk 3]
```diff
@@ -1,5 +1,14 @@
 a.nav-link {
-   padding: .5rem;
+   padding: .4rem;
+   /* font-family: 'Pinyon Script', cursive; */
```
Comment (JTHall94) [inline src/Components/NavigationCard.css:3]: If this isn't needed, you might want to take this out. Seems like you're opting for the different font-style below.

File: src/App.js [hunk 4]
```diff
@@ -40,17 +47,20 @@ class App extends React.Component {
...
       <div className="App">
         <Navbar />
         <Jumbotron jumboImg="trattoria.png" />
-        <NavigationCard description={descriptions[0].description} />
+        <NavigationCard menuItems={getMenu} />
+      </div>
+    ) :
+      <div class="d-flex justify-content-center">
+        <div className="spinner-border" role="status">
+          <span className="sr-only">Loading...</span>
```
Comment (JTHall94) [inline src/App.js:61]: This is great for accessibility, nice.

**Review by JTHall94 — APPROVED** [review]
This is really slick! You've done some great work in here to make everything more efficient and to dynamically generate content based on the information you're providing in the menu.json file. There are just a couple of areas with commented out code to look at in terms of cleaning up.

File: src/Components/Card.js (hunk 1 above)
Comment (toddtrivisonno) [reply src/Components/Card.js:11]: Thanks for all the feedback Jacob! I'll make the changes and have another pull request (or two) today.

**Pull request merged**